src/content/docs/tutorial-create-alerts/create-nrql-alerts.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "Create a NRQL alert"
+metaDescription: "How to create a NRQL alert with New Relic"
+---
+
+import alertsNRQL from 'images/alerts_screenshot-crop_NRQL-query.webp'
+
+import alertsNewCreate from 'images/alerts_screenshot-crop_create-new-query.webp'
+
```

Comment:
This doc shares a lot of content with the chart alerts docs—so much so I almost wonder if they should be combined, or at least acknowledge the overlap more explicitly.